src/AppBundle/EventSubscriber/PullRequestSubscriber.php

```diff
@@ -93,6 +95,24 @@ public function initPullRequestTypeLabel(GitHubEvent $githubEvent)
         ]);
     }
 
+    /**
+     * Add BC break label to the pull request according to the PR template.
+     *
+     * @param GitHubEvent $githubEvent
+     */
+    public function initBCBreakLabel(GitHubEvent $githubEvent)
+    {
+        $bcBreak = $this->issuesListener->addBackwardCompatibleLabel($githubEvent->getPullRequest());
+
+        $eventStatus = 'opened' === $githubEvent->getEvent()->getAction() ? 'opened' : 'edited';
+
+        $githubEvent->addStatus([
+            'event' => 'pr_'.$eventStatus,
+            'action' => 'BC break label initialized',
+            'status' => $bcBreak ? 'BC break' : 'no BC break',
```

Comment:
Is there a reason to have string as `status`, since it concerns the BC break, we can use a bool wdyt?

Reply:
Definitely!